lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -125,7 +125,7 @@ export class AuthorizationCodeClient extends BaseClient {
         }
         const queryString = this.createLogoutUrlQueryString(logoutRequest);
 
-        // Construct logout URI.
+        // Construct logout URI
```

Comment:
@hectormmg are you using this branch just to test out your CI pipeline changes?  If so, maybe make a separate PR just for that?

Reply:
@shylasummers and I were looking at the CI pipeline issue together since we first realized it was there when she created this PR. We pulled my branch into this one to see if the CI fixes were actually working, but next time I'll make sure it's on a completely separate branch.